packer/registry/image/fromartifact_example_test.go

```diff
@@ -55,7 +55,19 @@ func ExampleWithProvider() {
 	hcimage, _ := image.FromArtifact(a, image.WithProvider("happycloud"), image.WithRegion("west"))
 	fmt.Printf("%#v", *hcimage)
 	// Output:
-	// image.Image{ImageID:"service-id-123", ProviderName:"happycloud", ProviderRegion:"west", Labels:map[string]string{}}
+	// image.Image{ImageID:"service-id-123", ProviderName:"happycloud", ProviderRegion:"west", Labels:map[string]string{}, SourceImageID:""}
+}
+
+func ExampleWithSourceImageImageID() {
```

Comment:
This tests is failing because it is looking for a function with the name `WithSourceImageImageID`. I would update the example function to be like the following.
 
```suggestion
func ExampleWithSourceID() {
```

Reply:
Thanks!